week_1/__tests__/Array.test.js

```diff
@@ -28,3 +33,35 @@ test("max_subarray works", () => {
 test("two_sum works", () => {
 	expect(twoSum([2, 7, 11, 15], 9)).toEqual([0, 1]);
 });
+
+test("majority_element works", () => {
```

Comment:
can we remove all of these unrelated tests? everything else looks good 😄

Reply:
I think that I removed them. I tried to do it from the "Files Changed" section. Did that work?